File: src/main/java/de/cloudypanda/main/deathtimer/config/DeathTimerConfigManager.java
package de.cloudypanda.main.deathtimer.config;

import de.cloudypanda.main.Huntcraft;
import de.cloudypanda.main.common.file.AbstractFileManager;

public class DeathTimerConfigManager extends AbstractFileManager<DeathTimerConfigModel> {

    public DeathTimerConfigManager(String fileName, Huntcraft huntcraft) {
        super(fileName, huntcraft, DeathTimerConfigModel.class);
    }
}

File: src/main/java/de/cloudypanda/main/deathtimer/config/DeathTimerConfigManager.kt
package de.cloudypanda.main.deathtimer.config;

import de.cloudypanda.main.Huntcraft;
import de.cloudypanda.main.common.config.AbstractFileManager;

class DeathTimerConfigManager(fileName: String, huntcraft: Huntcraft) :
    AbstractFileManager<DeathTimerConfigModel>(fileName, huntcraft, DeathTimerConfigModel::class.java) {
}
